deploy/ci-test-vm/README.md

```diff
@@ -0,0 +1,114 @@
+## Overview
+
+A test/staging environment for Github Actions TLOmodel workflows.
+
+The VM has two cores, and we setup two runners for each core to reflect how runners are deployed in live environment.
+
+## Setup
+
+We will deploy the runners on Azure virtual machines, but you can also use Vagrant to create virtual machines on your own computer for local testing.
+
+### Using Vagrant for local testing
+
+- Fork TLOmodel to your personal account - this is where you'll install the runner.
+- Install Vagrant (requires VirtualBox)
+- Install Ansible
+ 
+You can use conda environment for Ansible:
+
+```sh
+conda create -n ansible python=3.8
+conda activate ansible
+conda install -c conda-forge ansible
+```
+
+Ansible logs in to the virtual machine using SSH. As you might make/destroy the VM many times, the guest fingerprint changes and then Ansible errors. To prevent this, set:
+
+```sh
+export ANSIBLE_HOST_KEY_CHECKING=False
+```
+
+To match the self-hosted running VMs deployed on Azure, we use the Ubuntu 22.04 LTS box:
+
+```sh
+vagrant up
+```
+
+Finally, provision the VM (uses Ansible):
+
+```sh
+vagrant provision
+```
```

Comment:
Vagrant VM will automatically get provisioned first time. This command useful to reprovision machine after editing playbook (at least, easier than the long ansible command!)

Reply:
Ah that's useful to know as that confused me initially.